build-in-container/create-build-container.sh

```diff
@@ -0,0 +1,4 @@
+#! /bin/bash
+set -e
+
+docker build -t 'mcr.microsoft.com/mariner-container-build:2.0' .
```

Comment:
This needs to handle not being in the correct directory.
```suggestion
( cd ${tool_dir} || { echo "ERROR: Could not change directory to ${tool_dir}; exit 1; } && \
docker build -t 'mcr.microsoft.com/mariner-container-build:2.0' . )
```

Reply:
Changed to docker build -t 'mcr.microsoft.com/mariner-container-build:2.0' ${tool_dir}